website/src/theme/MDXComponents/A.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
please set up `prettier` and `eslint` locally and run them over your files. Those are not properly formatted files.

Reply:
?? are you referring to the .js files or .md files?

the .js files in MDXComponents have been auto generated with exception being index, video and videocontainer

Reply:
how are they autogenerated? by what?
even so, they should then be reformatted by a linter to match the coding standard.